api/src/controllers/users.js

```diff
@@ -143,7 +145,7 @@ function UsersControllerFactory (sequelize, auth, User, Invite, log, ledger, soc
 
       let dbUser
       let invite
-      yield sequelize.transaction(sequelize.Promise.coroutine(function * (t) {
+      yield sequelize.transaction(co.wrap(function * (t) {
```

Comment:
Any reason why you change this to `co.wrap`? Wasn't the goal of the refactor to get ride of `co`?